### ISSUE TITLE
Una justificación medio rara

### ISSUE THREAD

**Issue by germanflighel** [ezecastiglione18/TallerMecanico#6]
https://github.com/ezecastiglione18/TallerMecanico/blob/e6aacc0c0296c5276d3f40020224d435427c9815/TallerMecanico.hs#L171-L175

Hay un par de cosas que están mal: la realidad no es que el `filter` no termina, sino que no se evalúa la expresión! Lo que si no termina el `sum`, ya que no termina de evaluar jamás al encontrarse con una lista infinita (para vos Análisis Matemático 1).
El filter sobre una lista infinita es una expresión válida al 100%, sólo hay que tener cuidado al trabajar con eso, porque sigue siendo una lista infinita. 

Se ve mejor?

**Comment by ezecastiglione18** [comment]
No, perdon jajajaj

Pero el tema es que la suma, para que te devuelva la sumatoria, necesita primero una lista finita. No podes hacer 2+4+6+8+10+12+14+...

**Comment by germanflighel** [comment]
Exacto, como decís! Pero eso no fue lo que escribieron jajaja, porque ustedes me dicen que el `filter` no termina de evaluar a todos los elementos de la lista, lo que es cierto si le pedís que evalúe! Pero la expresión es válida, me va a retornar la lista infinita! Y esa es la diferencia con el `sum`: este último se cuelga, no me devuelve nada, porque no lo puede hacer!
